examples/basics/visuals/implicitmesh.py

```diff
@@ -0,0 +1,54 @@
+"""
+Demonstration of an implicitly connected mesh.
+"""
+
+import sys
+from vispy import scene
+from vispy.color import get_colormap
+
+import numpy as np
+from scipy.special import sph_harm
```

Comment:
This `scipy` import is what's breaking the Travis build. Can you use `git grep` to find the other example where `scipy` is used? If we skip that example, please also skip this one. I think we usually also add some docstring at the top saying what the non-standard dependencies are, like `scipy`.